### PR TITLE
[gbp no-update]fix apc as anything runtiming

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
fix issue brought up in #66186 thanks to @SabreML

## Why It's Good For The Game
runtime issue fix
image

### CODE REVIEW

**Comment by cacogen** [comment]
breaked 😔

File: code/modules/power/apc/apc_power_proc.dm [hunk 1]
```diff
@@ -110,11 +110,11 @@
 		terminal = null
 
 /obj/machinery/power/apc/proc/energy_fail(duration)
-	for(var/obj/machinery/failing_machine as anything in area.contents)
+	for(var/obj/machinery/failing_machine in area.contents)
 		if(failing_machine.critical_machine)
 			return
 
-	for(var/mob/living/silicon/ai as anything in GLOB.ai_list)
```
Comment (Fikou) [inline code/modules/power/apc/apc_power_proc.dm:117]: why

Comment (Fikou) [reply code/modules/power/apc/apc_power_proc.dm:117]: ai list will always be ais

Comment (Ghilker) [reply code/modules/power/apc/apc_power_proc.dm:117]: True

File: code/modules/power/apc/apc_power_proc.dm [hunk 2]
```diff
@@ -110,11 +110,11 @@
...
+	for(var/mob/living/silicon/ai in GLOB.ai_list)
```
Comment (Ghilker) [inline code/modules/power/apc/apc_power_proc.dm:117]:
```suggestion
	for(var/mob/living/silicon/ai as anything in GLOB.ai_list)
```

**Pull request merged**